src/service/resources-impl.js

```diff
@@ -794,7 +794,10 @@ export class Resources {
             minTop = minTop == -1 ? box.top : Math.min(minTop, box.top);
           }
           request.resource./*OK*/changeSize(
-              request.newHeight, request.newWidth, request.callback);
+              request.newHeight, request.newWidth);
+          if (request.callback) {
+            request.callback(/* hasSizeChanged */true);
```

Comment:
It'll be [resolved](https://github.com/ampproject/amphtml/pull/4015/files#diff-80b9cea934a00a64f15d14cb656dfad4R825) after the scroll.